ansible/vars.yaml

```diff
@@ -53,6 +53,9 @@ doctor_visits_midas_password: "{{ vault_doctor_visits_midas_password }}"
 # NCHS
 nchs_mortality_token: "{{ vault_nchs_mortality_token }}"
 
+# NWSS
+nwss_mortality_token: "{{ vault_nchs_mortality_token }}"
```

Comment:
maybe we can eventually change `vault_nchs_mortality_token` to `vault_socrata_token` or perhaps `vault_cdc_socrata_token`, but thatll need help from someone who knows how to do the vault stuff...  @minhkhul is that difficult?

Reply:
Nope not difficult. Will do that change after   this indicator ships though, since it involve another repo.

Reply:
I was wrong! The secrets _are_ in this repo. Will make changes shortly.

Reply:
Changes are ready for review.

Reply:
i (perhaps obviously) dont know whats going on with the `vault.yaml` changes, but the rest of it looks good to me!